phi.float/src/datapack/data/phi.float/functions/internal/add_resume.mcfunction

```diff
@@ -0,0 +1,16 @@
+# Move inputs to usable positions for adding
+data modify entity @s ArmorItems[0].tag.AttributeModifiers[0].Amount set from block -30000000 0 1600 RecordItem.tag.phi.float.inputA
+data modify entity @s ArmorItems[1].tag.AttributeModifiers[0].Amount set from block -30000000 0 1600 RecordItem.tag.phi.float.inputB
+
+# Since the health might have gone down, raise it back up
+data modify entity @s Health set value 1024f
```

Comment:
Would this still be able to handle sums larger than 1024?

Reply:
`Health` is hardcapped at 1024 and, since it's where the output is being pulled from, the 1024 cap is here to stay. Now, if we can grab negative values... they would be unbounded.

Reply:
Just tested it. Health is also hardcapped to be above 0. I really had my hopes up, there.